run_benchmarks.rb

```diff
@@ -74,6 +75,13 @@ def set_bench_config(turbo:)
     end
     check_call("sudo -S cpupower frequency-set -g performance") unless performance_governor?
   end
+
+  if os == :linux && ENV["BENCHMARK_MODE"] != '0'
```

Comment:
Sure, we can make this an arg.
It is currently "on" by default, and doesn't do anything if it can't, but I wanted to be able to disable it for comparison.

Reply:
Ok makes sense